packages/osc-recruitment/app/root.tsx

```diff
@@ -1,5 +1,4 @@
-import { cssBundleHref } from '@remix-run/css-bundle';
-import { json, LoaderFunction, type LinksFunction } from '@remix-run/node';
+import { LoaderFunction, json, type LinksFunction } from '@remix-run/node';
```

Comment:
Another eslint warning here that I think Prettier should be able to solve, if not just change the import to 
```suggestion
import { LoaderFunction, json } from '@remix-run/node';
import type { LinksFunction } from '@remix-run/node';
```